contracts/UniswapV2Factory.sol

```diff
@@ -56,4 +56,8 @@ contract UniswapV2Factory is IUniswapV2Factory {
         require(msg.sender == feeToSetter, "UniswapV2: FORBIDDEN");
         feeToSetter = _feeToSetter;
     }
+    function getPairHash() external returns (bytes32) {
```

Comment:
I think it should be pure function.

It changes UniswapV2Fatory, Could you add an uint test for this?

Reply:
It's better to add this to the top of the file as a constant:

`bytes32 public constant PAIR_HASH = keccak256(abi.encodePacked(type(GenesisPair).creationCode));`